src/ui/components/NetworkMonitor/RequestTable.tsx

```diff
@@ -30,14 +42,65 @@ const RequestTable = ({
 }) => {
   const { columns, getTableProps, getTableBodyProps, headerGroups, rows, prepareRow } = table;
 
+  const dispatch = useDispatch();
```

Comment:
:hand: FYI this actually suffers from the same `dispatch` type problem as above, and we really _should_ be using the "typed hooks" solution I linked in the other comment.  But, there's a quirk in the React-Redux v7.x types that lets you sorta get away with dispatching thunks here, because it's really accepting `any` somehow.  But ideally this would be `const dispatch = useAppDispatch()` with the right type baked in.